rails_routes_api_engine.gemspec

```diff
@@ -14,9 +14,16 @@ Gem::Specification.new do |s|
   s.description = "Exposes your Rails app's routes via an API endpoint"
   s.license     = "MIT"
 
-  s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
+  s.files = Dir[
+    "{app,config,db,lib}/**/*",
+    "MIT-LICENSE",
```

Comment:
After re-reading the MIT License, I don't think it's the right for the company. Mainly, this part:

```Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software```